src/definitions/elements/icon.less

```
@@ -376,6 +376,8 @@ i.icons {
     display: inline-block;
     position: relative;
     line-height: 1;
+    min-width: @width;
+    min-height: @height;
```

Good catch. 
We should also add the same margin and text-align as a single icon, so there definately is no visual difference anymore
See https://jsfiddle.net/lubber/axyf20j5/8/
```suggestion
    min-width: @width;
    min-height: @height;
    margin: 0 @distanceFromText 0 0;
    text-align: center;
```

---

Changes applied